MoreLinq/TrySingle.cs

```diff
@@ -0,0 +1,58 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2019 James Webster. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+namespace MoreLinq
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    partial class MoreEnumerable
+    {
+        /// <summary>
+        /// Similar to Single() or SingleOrDefault() but allows the caller to determine
```

Comment:
Summaries should not refer to other methods. If they do, use `<see cref="..." />` to call them out.

Reply:
Yeah, I normally do that but was getting a compiler error. I'm sure the imports were valid as well...